commpy/links.py

```diff
@@ -200,18 +203,21 @@ def link_performance_full_metrics(self, SNRs, tx_max, err_min, send_chunk=None,
         BEs = np.zeros((len(SNRs), tx_max), dtype=int)  # Bit errors per tx
         CEs = np.zeros((len(SNRs), tx_max), dtype=int)  # Chunk Errors per tx
         NCs = np.zeros((len(SNRs), tx_max), dtype=int)  # Number of Chunks per tx
-        # Set chunk size and round it to be a multiple of num_bits_symbol*nb_tx to avoid padding
+        # Set chunk size and round it to be a multiple of num_bits_symbol* nb_tx to avoid padding taking in to account the coding rate
         if send_chunk is None:
             send_chunk = err_min
-        divider = self.num_bits_symbol * self.channel.nb_tx
```

Comment:
Ok. Then, it seems to me that changing this line with the following should fixe #85 without requiering Fraction. Are there, rounding errors in this line?

`divider = self.num_bits_symbol * self.channel.nb_tx / code_rate`

Reply:
That line (and/or the next one) is indeed the centre of the question, and where it needs "the fix".
But that change is not the same that I'm suggesting. That finds the send chunk that when applied the coding rate is divisible by num_bits_symbol, but does not ensure in the receiver, after demodulation, the number of bits before decoding are divisible by the code_rate.

Taking the example of my previous comment, the issue would still exist.
The result send_chunk  with that code would be `(600//(6/(3/4)))*6/(3/4) = 600`under the code that I have suggested, the result would be `594 = (600//(6*3)) * (6*3)`, 594 after coding 3/4 will be 792 which is divisible by 6.